tests/test_encrypted_cookie_storage.py

```diff
@@ -5,69 +5,94 @@
 
 import pytest
 from aiohttp import web
+from aiohttp.web_middlewares import _Handler
+from aiohttp.test_utils import TestClient
+
+from typing import no_type_check, Any, Dict, Tuple, Union
 
 from cryptography.fernet import Fernet
 
 from aiohttp_session import (Session, session_middleware, get_session,
                              new_session)
 from aiohttp_session.cookie_storage import EncryptedCookieStorage
 
+from typedefs import _TAiohttpClient
+
 
 MAX_AGE = 1
 
 
-def make_cookie(client, fernet, data):
+def make_cookie(
+    client: TestClient,
+    fernet: Fernet,
+    data: Dict[Any, Any]
+) -> None:
     session_data = {
         'session': data,
         'created': int(time.time())
     }
 
     cookie_data = json.dumps(session_data).encode('utf-8')
-    data = fernet.encrypt(cookie_data).decode('utf-8')
+    encrypted_data = fernet.encrypt(cookie_data).decode('utf-8')  # type: str
 
-    client.session.cookie_jar.update_cookies({'AIOHTTP_SESSION': data})
+    # Ignoring type until aiohttp#4252 is released
+    client.session.cookie_jar.update_cookies(
+        {'AIOHTTP_SESSION': encrypted_data}  # type: ignore
+    )
 
 
-def create_app(handler, key):
+def create_app(
+    handler: _Handler,
+    key: Union[str, bytes, bytearray]
+) -> web.Application:
     middleware = session_middleware(EncryptedCookieStorage(key))
     app = web.Application(middlewares=[middleware])
     app.router.add_route('GET', '/', handler)
     return app
 
 
-def decrypt(fernet, cookie_value):
+def decrypt(fernet: Fernet, cookie_value: str) -> Any:
     assert type(cookie_value) == str
     return json.loads(
         fernet.decrypt(cookie_value.encode('utf-8')).decode('utf-8')
     )
 
 
+# pytest.fixture decorator strips the typing of the decorated function
+@no_type_check
```

Comment:
Shouldn't be using this. This is used to flag that annotations are not meant to be type definitions.

I see there is typing support in pytest, so maybe this has already been fixed by them. Otherwise, let's figure out another solution.